src/lamplib/tasks/toolchain.py

```diff
@@ -52,10 +51,11 @@ def fetch_and_install_toolchain(url, install_dir):
             logging.info('Finished Downloading gennytoolchain as %s', toolchain_tarball)
 
         logging.info('Extracting gennytoolchain into %s, please wait...', toolchain_dir)
-        tarball = tarfile.open(toolchain_tarball)
 
         shutil.rmtree(toolchain_dir, ignore_errors=True)
```

Comment:
I'd be curious if we stopped ignoring errors whether we'd find `tarball.extractall()` is failing to overwrite existing files (or symlinks).

Reply:
🤷‍♂ fwiw we do [clean out previous toolchain installs](https://github.com/mongodb/genny/blob/7914a179e3fcc403ba09174df965ad0b74717939/evergreen.yml#L382)